fix for "Segmentation fault" in nsupdate

 $ nsupdate
 > update
 Segmentation fault

from https://bugs.archlinux.org/task/30139?getfile=8868
(they say: "The upstream bug reference number is 29550")
--- /home/bisson/Desktop/bind-9.9.1-P1/bin/nsupdate/nsupdate.c  2012-06-02 01:29:52.000000000 +1000
+++ bin/nsupdate/nsupdate.c     2012-06-06 02:50:37.894357220 +1000
@@ -376,7 +376,7 @@
 	char sc, dc;
 
 	if (string == NULL)
-		return (NULL);
+		return &string;
 
 	for (; *string != '\0'; string++) {
 		sc = *string;
